serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -220,71 +223,71 @@ Learn how to use the OpenAPI Callback in your workflow application
 [.card]
 --
 [.card-title]
-xref:security/authention-support-for-openapi-services.adoc[Authentication for OpenAPI services in {product_name}]
+xref:security/authention-support-for-openapi-services.adoc[]
 [.card-description]
 Learn how to use authentication methods when calling REST services using OpenAPI specification
 --
 
 [.card]
 --
 [.card-title]
-xref:security/orchestrating-third-party-services-with-oauth2.adoc[Orchestration of third-party services using OAuth 2.0 authentication]
+xref:security/orchestrating-third-party-services-with-oauth2.adoc[]
 [.card-description]
 Learn about the OAuth2 method support when orchestrating REST services using your workflow application
 --
 
 [.card-section]
-== Testing and Troubleshooting
+== Executing, Testing and Troubleshooting
 
 [.card]
 --
 [.card-title]
-xref:testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc[Testing your workflow application using REST Assured]
+xref:testing-and-troubleshooting/mocking-openapi-services-with-wiremock.adoc[]
```

Comment:
mocking openapi card should be after mocking http events one (similar to nav bar)

Reply:
Sure, the devil lies in the details. 😈